source/extensions/filters/http/transformation/transformation_filter.cc

```diff
@@ -191,7 +203,7 @@ void TransformationFilter::transformSomething(
     void (TransformationFilter::*addData)(Buffer::Instance &)) {
 
   try {
-    transformation->transform(header_map, body, callbacks);
+    transformation->transform(header_map, request_headers_ , body, callbacks);
```

Comment:
rename `header_map` to `response_header_map`?

Reply:
header map can actually be either response or request